src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -532,25 +536,33 @@ CameraTrigger::cycle_trampoline(void *arg)
 	// Command handling
 	if (updated) {
 
+		hrt_abstime now = hrt_absolute_time();
+
 		orb_copy(ORB_ID(vehicle_command), trig->_command_sub, &cmd);
 
 		if (cmd.command == vehicle_command_s::VEHICLE_CMD_DO_DIGICAM_CONTROL) {
 
 			need_ack = true;
 
-			if (commandParamToInt(cmd.param7) == 1) {
-				// test shots are not logged or forwarded to GCS for geotagging
-				trig->_test_shot = true;
+			if (now - trig->_trigger_timestamp < trig->_min_interval * 1000) {
```

Comment:
What about using `hrt_elapsed_time()` to make it more readable?

Reply:
That call locked historically interrupts. Now it is an option.